BeVolt/App/src/main.c

```diff
@@ -93,6 +96,7 @@ void initialize(void){
 	Voltage_Init(Minions);
 	Temperature_Init(Minions);
 	CLI_Init(Minions);
+	CAN1_Init(CAN_Mode_Normal);
 
```

Comment:
Can you add your initialization function to initialize() so that everything your functions need will be initialized when the BPS is run during the race?